crowbar_framework/db/migrate/20160215120522_add_migration_level_to_backups.rb

```diff
@@ -0,0 +1,5 @@
+class AddMigrationLevelToBackups < ActiveRecord::Migration
+  def change
+    add_column :backups, :migration_level, :integer, limit: 8
```

Comment:
this can also be changed in another migration if needed with `change_column_default` in the future